src/main/java/com/flightstats/hub/config/properties/SpokeProperties.java

```diff
@@ -27,8 +29,8 @@ public int getTtlMinutes(SpokeStore spokeStore) {
     public String getPath(SpokeStore spokeStore) {
         String property = "spoke." + spokeStore + ".path";
         String fallbackProperty = "spoke.path";
-        String defaultPath = "/spoke/" + spokeStore;
-        return propertiesLoader.getProperty(property, propertiesLoader.getProperty(fallbackProperty, defaultPath));
+        String defaultPath = Paths.get(FileSystems.getDefault().getSeparator(),"spoke", String.valueOf(spokeStore)).toString();
```

Comment:
"spoke" - Instead of hardcoding string here, can this also be accessed using a variable from  SpokePathUtil class ?

Reply:
Since this file belongs to load the spoke properties value from ***hub.properties file or default value if not present , we are mentioning(hardcoding) all the keys over here. But the corresponding values are picked from the properties file. So am leaving this as it is.